Gemfile.lock

```diff
@@ -257,8 +256,8 @@ GEM
       rack (>= 1.4, < 3)
       rack-protection (>= 1.5.3, < 2.2.0)
     formatador (0.2.5)
-    formtastic (3.1.5)
-      actionpack (>= 3.2.13)
+    formtastic (4.0.0)
```

Comment:
version bump necessary to fix this issue: https://github.com/formtastic/formtastic/issues/ 1325